File: src/main/java/de/scandio/e4/virtualusers/VirtualUserB.java
package de.scandio.e4.virtualusers;

import de.scandio.e4.interfaces.Scenario;
import de.scandio.e4.interfaces.VirtualUser;

import java.util.List;

public class VirtualUserB implements VirtualUser {
    @Override
    public List<Class<Scenario>> getScenarios() {
        return null;
    }
}

File: src/main/kotlin/de/scandio/e4/virtualusers/VirtualUserB.kt
package de.scandio.e4.virtualusers

import de.scandio.e4.worker.interfaces.Scenario
import de.scandio.e4.worker.interfaces.VirtualUser

class VirtualUserB : VirtualUser {
    override fun getScenarios(): List<Scenario> {
        val list = arrayListOf<Scenario>()
        return list
    }
}